### PR TITLE
libparserutils@0.2.5 (#2478)

### PR DESCRIPTION
**Pull request by dmitry-j-mikhin**
PR for #2478

Release: https://source.netsurf-browser.org/libparserutils.git/tag/?h=release/0.2.5

Used by:
* https://www.netsurf-browser.org/

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (libparserutils) have been updated in this PR. Please review the changes.

File: modules/libparserutils/0.2.5/patches/add_build_file.patch
Comment (Wyverald) [inline modules/libparserutils/0.2.5/patches/add_build_file.patch:1]: did this BUILD file come from somewhere? if so, could you add a comment here?

File: modules/libparserutils/0.2.5/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@libparserutils//:libperserutils'
```
Comment (Wyverald) [inline modules/libparserutils/0.2.5/presubmit.yml:17]:
```suggestion
    - '@libparserutils//:libparserutils'
```

File: modules/libparserutils/0.2.5/patches/add_build_file.patch [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
++        "src/input/inputstream.c",
++    ],
++    hdrs = _LIBPARSERUTILS_PUB_HEADERS,
++    includes = ["src", "include"],
++    copts = ["-w"],
++    visibility = ["//visibility:private"],
++)
++
++cc_library(
++    name = "libparserutils",
```
Comment (Wyverald) [inline modules/libparserutils/0.2.5/patches/add_build_file.patch:57]: consider doing the same thing as https://github.com/bazelbuild/bazel-central-registry/pull/1005#discussion_r1359865046

File: modules/libparserutils/0.2.5/presubmit.yml (hunk 1 above)
Comment (dmitry-j-mikhin) [reply modules/libparserutils/0.2.5/presubmit.yml:17]: My bad, thanks.

File: modules/libparserutils/0.2.5/patches/add_build_file.patch (hunk 2 above)
Comment (dmitry-j-mikhin) [reply modules/libparserutils/0.2.5/patches/add_build_file.patch:57]: My bad, thanks.

File: modules/libparserutils/0.2.5/patches/add_build_file.patch
Comment (dmitry-j-mikhin) [reply modules/libparserutils/0.2.5/patches/add_build_file.patch:1]: I searched internet and couldn't find a similar receipt, so I did it myself.

Comment (Wyverald) [reply modules/libparserutils/0.2.5/patches/add_build_file.patch:1]: would you mind adding yourself as a module maintainer in metadata.json, then? :)

Comment (dmitry-j-mikhin) [reply modules/libparserutils/0.2.5/patches/add_build_file.patch:1]: Ok, I will do :)

**Review by Wyverald — APPROVED** [review]

**Pull request merged**